lib/dal-test/src/lib.rs

```diff
@@ -3,6 +3,9 @@
 
 #![warn(
     bad_style,
+    clippy::expect_used,
+    clippy::panic,
+    clippy::unwrap_used,
```

Comment:
No panics by default in `dal-test` other than in macro expansion helpers (`expand_helpers`) and in one-off scenarios. This includes test exclusive schema scaffolding as well as all other kinds of helpers.